back/courses/templates/courses/view_course.html

```diff
@@ -0,0 +1,103 @@
+{% load static %}
+<!--Markdownify-->
+{% load markdownify %}
+<!--Header-->
+{% if course.acronym %}{% include "header.html" with title=course.acronym %}{% else %}{% include "header.html" with title=course.name %}{% endif %}
+
+<!--Body-->
+<body>
+    {% comment %} <!--Feedback Button-->
+    {% include "feedback.html" %} {% endcomment %}
+
+    <!--Sidenav-->
+    {% include "sidenav.html" with active_link="courses" %}
+
+    <!--Content-->
+    <div class="content">
+
+        <!--Header Content-->
+        <div class="content-header">
+            <div class="row justify-content-between w-100">
+                <div class="col">
+                    <div class="search-input-group">
+                        <input type="text" class="search-input" placeholder="Rechercher">
+                        <button class="button search-button"><i class="fas fa-search "></i></button>
+                    </div>
+                </div>
+            </div>
+        </div>
```

Comment:
Remove the search bar here, it is useless

Reply:
I missed it. I should be more carefull with copy/paste !